en/setup/vscode.md

```diff
@@ -15,6 +12,9 @@ This topic explains how to setup the IDE and start developing.
 > **Note** There are other powerful IDEs, but they typically take more effort to integrate with PX4.
   With *VScode*, configuration is stored in the PX4/Firmware tree ([Firmware/.vscode](https://github.com/PX4/Firmware/tree/master/.vscode)) so the setup process is as simple as adding the project folder.
 
+<span></span>
+> **Tip** On Ubuntu 16.04, VSCode can be used to build PX4 but not debug targets.
```

Comment:
Could you add: [...] We therefore recommend using Ubuntu 18.04 instead. (or just deleting the comment). The way its put right now it sounds like 16.04 is the supported target but has deficiencies.

Reply:
Yes, decided to remove. We already suggest that this works on 18.04 above.